translation/source/challenge.xml

```diff
@@ -22,5 +22,5 @@
   <string name="declineVariant">I'm not willing to play this variant right now.</string>
   <string name="declineNoBot">I'm not accepting challenges from bots.</string>
   <string name="declineOnlyBot">I'm only accepting challenges from bots.</string>
-  <string name="inviteLichessUser">Or invite a Lichess User:</string>
+  <string name="inviteLichessUser">or, invite a Lichess user:</string>
```

Comment:
This one I think lowercase "user" is good, but I'd say capitalized "Or" and no comma.

![image](https://github.com/lichess-org/lila/assets/271432/baa33c78-9563-4469-90c0-26c17b8166a5)
![image](https://github.com/lichess-org/lila/assets/271432/8e8a29d2-ee33-47cc-acce-e3c987d9d198)